tools/klee-replay/file-creator.c

```diff
@@ -149,7 +150,7 @@ static int create_char_dev(const char *fname, exe_disk_file_t *dfile,
     }
 
     if (symlink(name, fname) == -1) {
-      fprintf(stderr, "unable to create sym link to tty\n");
+      fprintf(stderr, "KLEE-REPLAY: ERROR: unable to create sym link to tty\n");
```

Comment:
All `fprintf`s without format string should be `fputs`.

Reply:
Why?

Reply:
fprintf is way more complex and burns CPU cycles for nothing.

Reply:
OK, but the difference doesn't matter in most cases (such as here, for error reporting) and nowadays many compilers do this type of optimization themselves.  The reason I prefer `(f)printf` slightly is that I can then use everywhere, regardless of whether it's a simple string or not.

Reply:
I don't feel strongly about this, I only think it's a bad habit to rely on a compiler optimisation. Why use a function with full-blown var-arg handling and string parsing for sth. that simple in the first place? Also it makes reading the code harder as one automatically tries to find format specifiers or arguments... but it's just my opinion and there are more important issues than ~20 misused `fprintf`s.

Reply:
OK, thanks for being flexible ;)   I could agree with this for an isolated instance, but in this case, we have a ton of `fprintf`s in there, and I find it less readable to switch from `fprintf`to `fputs` or vice-versa.  For instance:
```
fprintf(stderr, "Usage: %s [option]... <executable> <ktest-file>...\n", progname);
fprintf(stderr, "   or: %s --create-files-only <ktest-file>\n", progname);
fprintf(stderr, "\n");
fprintf(stderr, "-r, --chroot-to-dir=DIR  use chroot jail, requires CAP_SYS_CHROOT\n");
```
the last two `fprintf`s could be replaced by `fputs`, but wouldn't you find it a bit ugly to do so?

Reply:
No, because here I would just use a single `fprintf` as I did in [gen-bout.c](https://github.com/klee/klee/blob/64508909bcbd52d6a06ec0f299b3752368d10e31/tools/gen-bout/gen-bout.cpp#L50). (The formatting is still weird, I kept the original one.)

Reply:
I see :)  But isn't it slower than mixing `fputs` and `fprintf`? ;) 
In any case, feel free to submit a patch if you want to do a pass over the way messages are printed in there.  A better option would be perhaps to offer functions such as `klee_replay_error()` etc. as we do in the core KLEE tool.